### DISCUSSION TITLE
Code available?

### DISCUSSION THREAD

**Discussion by cheffe112** [empriselab/RCareWorld#3] [Help - Using] [answered]
Hello, I'm interested into having a look into your code. This looks very promising for a number of possible scenarios.
Could you tell me what your plans are regarding publishing the code here? Thanks!

**Comment by aty8** [comment]
Hi, any updates? Is the project alive?

**Comment by YoruCathy** [comment, marked as answer]
Hi, yes we will release the code here in early March, as is shown on our website. Stay tuned!
